src/routers/sra_router.ts

```diff
@@ -45,6 +45,12 @@ export function createSRARouter(orderBook: OrderBookService): express.Router {
      * This is an additional endpoint not a part of the official SRA standard
      */
     router.post('/orders', asyncHandler(handlers.postOrdersAsync.bind(handlers)));
+    /**
+     * POST Persistent order endpoint submits an order that will be persisted even after cancellation or expiration.
+     * Requires an API Key
+     * This is an additional endpoint not a part of the official SRA standard
+     */
+    router.post('/order/persistent', asyncHandler(handlers.postPersistentOrderAsync.bind(handlers)));
```

Comment:
what are your thoughts on this custom endpoint versus just using `POST /order` with an API Key, iff there is an API key we also save it to `PersistentOrders`.